### DISCUSSION TITLE
Unexpected identifier "funcValidateion"

### DISCUSSION THREAD

**Discussion by yuxxeun** [livewire/livewire#7562] [1. Help]
I got this type of error, i dont know why this happen, and also i was trying to search on internet but got nothing, anyone can help me?

![image](https://github.com/livewire/livewire/assets/50067865/14388609-7d60-4b7e-a0b7-211fc4c26ae2)

**Comment by joshhanley** [comment]
@yuxxeun looking like you are missing a `function` from in front of one of your method names. But hard to say which without seeing your code.

**Reply by yuxxeun** [reply]
oops sorry my bad, this is actually on path `/something/create` and this is a store function that related to that path
![image](https://github.com/livewire/livewire/assets/50067865/bdd4e5b2-ec1b-41e7-bbfc-c00855edd291)

**Reply by joshhanley** [reply]
@yuxxeun can you share a screenshot that include the full stack trace?
